g/godogv12/godogubi85.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: godog
+# Version	: v0.12.0
+# Source repo	: https://github.com/cucumber/godog
+# Tested on	: ubi 8.5
+# Language      : go
+# Travis-Check  : true 
+# Script License: Apache License, Version 2 or later
+# Maintainer	: BulkPackageSearch Automation {maintainer}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=godog
+PACKAGE_VERSION=v0.12.0
+PACKAGE_URL=https://github.com/cucumber/godog.git
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
+
+# Install Go and setup working directory
+wget https://golang.org/dl/go1.16.1.linux-ppc64le.tar.gz && \
+    tar -C /bin -xf go1.16.1.linux-ppc64le.tar.gz && \
+    mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
+
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/tester/go
+
+export PATH=$GOPATH/bin:$PATH
+export GO111MODULE=on
+
+git clone $PACKAGE_URL
+
+cd godog
+
+git checkout $PACKAGE_VERSION
+
+if ! go test ./...; then
+		        echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+		        echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_fails
+		        echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | master  | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails" > /home/tester/output/version_tracker
```

Comment:
Created directories under `/home/tester/go/`, there is no **output** directory available there.